rotkehlchen/tests/utils/blockchain.py

```diff
@@ -221,21 +221,38 @@ def mock_etherscan_query(
     original_queries = [] if original_queries is None else original_queries
     extra_flags = [] if extra_flags is None else extra_flags
 
-    def mock_requests_get(url, *args, **kwargs):
-        if 'etherscan.io/api?module=account&action=balance&address' in url:
+    def mock_requests_get(url, params, *args, **kwargs):
+
+        def check_params(
+                values: dict[str, str] | None = None,
+                keys: list[str] | None = None,
+        ) -> bool:
+            if values is not None:
+                for key, value in values.items():
+                    if params.get(key) != value:
+                        return False
+
+            if keys is not None:
+                for key in keys:
+                    if key not in params:
+                        return False
+
+            return True
+
+        if 'etherscan.io/api' in url and check_params(values={'module': 'account', 'action': 'balance'}, keys=['address']):  # noqa: E501
```

Comment:
same here for etherscan api. Use the same constant all across the file/s.

Reply:
Using constant now, but actually moved the url check into the `check_params` function to cut down on repetition even more.